.github/workflows/terraform-destroy-dev-environments-cron.yml

```diff
@@ -10,13 +10,29 @@ permissions:
   contents: read  # This is required for actions/checkout
     
 jobs:
+  cleanup_versions_process:
+    name: Cleanup Versions Process
+    uses: ./.github/workflows/cleanup-appconfig-and-lambda-layer-versions.yml
+    strategy:
+      matrix:
+        # Update with new sandbox environments here
+        sandbox-name: [ ndra, ndrb, ndrc ]
```

Comment:
Why are we excluding ndrd?

Reply:
I didn't get chance to discuss this with Scott last Friday, it had been excluded from the matrix on line 36 also also which makes me think ndrd was been kept aside for demo purposes. So for now just keeping consistency with the other matrixes and when confirmed we can add ndrd back in